frontend/components/commons/header/filters/FiltersList.vue

```diff
@@ -290,6 +299,20 @@ export default {
       this.$emit("applySortBy", sortList);
       this.close();
     },
+    sortByDateFilter(id, name) {
+      return {
+        id: id,
+        key: id,
+        group: "Sort",
+        name: name,
+        disabled: this.recordPropertyHasValue(id),
+      };
```

Comment:
1/ shortcut: in es6, there is a shortcut in case the key and the value from object are the same => you can write return {id, key: id, group: "Sort", name, disabled: this.recordPropertyHasValue(id),}